### PR TITLE
Feature/cleanup code

### PR DESCRIPTION
**Pull request by Finka95**
Cleanup Code

### CODE REVIEW

**Review by Genresal — APPROVED** [review]

File: TicketSystem/TicketSystem/ViewModels/Messages/ShortMessage.cs [hunk 1]
```diff
@@ -1,9 +1,6 @@
-﻿using TicketSystem.ViewModels.Users;
+﻿namespace TicketSystem.ViewModels.Messages;
 
-namespace TicketSystem.ViewModels.Messages
+public class ShortMessage
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/Messages/ShortMessage.cs]: Rename to ShortMessageViewModel

File: TicketSystem/TicketSystem/ViewModels/Tickets/TicketVm.cs [hunk 2]
```diff
@@ -2,16 +2,15 @@
 using TicketSystem.ViewModels.Messages;
 using TicketSystem.ViewModels.Users;
 
-namespace TicketSystem.ViewModels.Tickets
+namespace TicketSystem.ViewModels.Tickets;
+
+public class TicketVm
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/Tickets/TicketVm.cs]: Same

File: TicketSystem/TicketSystem/ViewModels/Messages/MessageVm.cs [hunk 3]
```diff
@@ -1,13 +1,12 @@
 ﻿using TicketSystem.ViewModels.Users;
 
-namespace TicketSystem.ViewModels.Messages
+namespace TicketSystem.ViewModels.Messages;
+
+public class MessageVm
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/Messages/MessageVm.cs]: Rename to MessageViewModel

File: TicketSystem/TicketSystem/ViewModels/UserRoles/UserRoleEntity.cs [hunk 4]
```diff
@@ -1,8 +1,7 @@
-﻿namespace TicketSystem.DAL.Entities
+﻿namespace TicketSystem.DAL.Entities;
+
+public class UserRoleVm
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/UserRoles/UserRoleEntity.cs]: Same, and rename file

File: TicketSystem/TicketSystem/ViewModels/Users/ShortUser.cs [hunk 5]
```diff
@@ -1,10 +1,7 @@
-﻿using TicketSystem.BLL.Models;
+﻿namespace TicketSystem.ViewModels.Users;
 
-namespace TicketSystem.ViewModels.Users
+public class ShortUser
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/Users/ShortUser.cs]: same

File: TicketSystem/TicketSystem/ViewModels/Users/UserVm.cs [hunk 6]
```diff
@@ -1,9 +1,8 @@
-﻿namespace TicketSystem.ViewModels.Users
+﻿namespace TicketSystem.ViewModels.Users;
+
+public class UserVm
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/ViewModels/Users/UserVm.cs]: same

**Review by 5Ka-me — APPROVED** [review]

File: TicketSystem/TicketSystem/ViewModels/UserRoles/UserRoleEntity.cs (hunk 4 above)
Comment (pavelGagarin0870) [reply TicketSystem/TicketSystem/ViewModels/UserRoles/UserRoleEntity.cs]: Also better to add not short view models, currently u have several cases when get vm missed

File: TicketSystem/TicketSystem/BLL/Services/Abstractions/IUserService.cs [hunk 7]
```diff
@@ -1,14 +1,13 @@
...
-        Task<UserModel?> DeleteUserAsync(int id, CancellationToken cancellationToken);
-        Task<UserModel?> GetNotBusyOperator(CancellationToken cancellationToken);
-    }
-}
+    Task<UserModel> AddUserAsync(UserModel userModel, CancellationToken cancellationToken);
+    Task<UserModel?> GetUserByIdAsync(int id, CancellationToken cancellationToken);
+    Task<IEnumerable<UserModel>> GetUsersAsync(CancellationToken cancellationToken);
+    Task<UserModel?> UpdateUserAsync(UserModel userModel, CancellationToken cancellationToken);
+    Task<UserModel?> DeleteUserAsync(int id, CancellationToken cancellationToken);
+    Task<UserModel?> GetNotBusyOperator(CancellationToken cancellationToken);
```
Comment (pavelGagarin0870) [inline TicketSystem/TicketSystem/BLL/Services/Abstractions/IUserService.cs]: Maybe better to rename to GetAvailableOperator

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**